src/Documentation/Getting-Started-With-Orleans/Grain-Persistence.md

```diff
@@ -134,6 +134,9 @@ A quick way to test this is to (see in the aforementioned the few commented line
 The ADO.NET persistence has functionality to version data and define arbitrary (de)serializers with arbitrary application rules and streaming, but currently
 there is no method to expose them to application code. More information in [ADO.NET Persistence Rationale](#ADONETPersistenceRationale).
 
+__NB for Microsoft SQL Server users:__
+For the ADO.NET Storage Provider to work with SQL Server, you need to add [System.Data.SqlClient](https://www.nuget.org/packages/System.Data.SqlClient/) NuGet package to your project.
```

Comment:
My bad, looks like there are some PostgreSQL tests in our repo (which we don't run in CI anyway, as we don't run MySql tests either, but they are still there). There is no subclass of [RelationalStorageTests](https://github.com/dotnet/orleans/blob/016c0752dec649274f2cf3f2535a662b97618b07/test/TesterSQLUtils/StorageTests/Relational/RelationalStorageTests.cs) which seems to be the newest tests that were added. That was the only thing I looked before when I said there were no tests for Postgre, because I saw only subclasses for SQL Server and MySql.